### PR TITLE
🥩 Implement WG average stake

### LINKED ISSUE #415: Display working group dashboard
Follow-up of [#289](https://github.com/Joystream/pioneer/issues/289).

Implement:

*   [x] current budget
*   [x] debt
*   [x] avg stake See: #394

As for now only, the "current budget" is described in the API. Waiting for other fields to be defined or implemented.

### PR DESCRIPTION
**Pull request by DevNikita16**
Closes #415

### CODE REVIEW

File: packages/ui/src/working-groups/model/getAverageStake.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import BN from 'bn.js'
+
+import { WorkerFieldsFragment } from '@/working-groups/queries'
+
+export const getAverageStake = (workers: Pick<WorkerFieldsFragment, 'stake'>[]): BN => {
+  const totalStake = workers.reduce((a, b) => a + b.stake, 0)
```
Comment (jodator) [inline packages/ui/src/working-groups/model/getAverageStake.ts:6]: Similarly to the other PR - you can use `BN` here.

File: packages/ui/src/working-groups/queries/workingGroups.graphql [hunk 2]
```diff
@@ -49,7 +49,7 @@ fragment WorkingGroupFields on WorkingGroup {
     ...WorkingGroupMetdataFields
   }
   workers {
-    id
+    stake
```
Comment (jodator) [inline packages/ui/src/working-groups/queries/workingGroups.graphql:52]: So we don't use workers ids anywhwere? IIRC we changed how the list of workers is fetched, so this makes sense. Just to double check.

**Review by jodator — CHANGES_REQUESTED** [review]
Just the BN thing to update and we can merge it.

File: packages/ui/src/working-groups/queries/workingGroups.graphql (hunk 2 above)
Comment (DevNikita16) [reply packages/ui/src/working-groups/queries/workingGroups.graphql:52]: Yep, we catch workers separately from working group now.
That's why having worker ids is useless now.

File: packages/ui/src/working-groups/model/getAverageStake.ts (hunk 1 above)
Comment (DevNikita16) [reply packages/ui/src/working-groups/model/getAverageStake.ts]: ✅

**Comment by jodator** [comment]
@DevNikita16 could you check merge conflicts as well?

**Review by jodator — APPROVED** [review]

**Pull request merged**